src/index.html

```diff
@@ -69,382 +69,379 @@
           alt="product-img"
```

Comment:
think of disabled people who can't see images
and your shop has more than one product, if each image regardless of the product will have `alt="product-img"` - not great
why not `alt="APPLE iMac 27 inches Retina 5K Monoblock"`?